src/test/java/plugins/GerritTriggerTest.java

```diff
@@ -204,9 +204,8 @@ private void checkApprovalValueFromJSON(String json, String labelName, int value
     }
 
     private File createGitCommit(String jobName) throws IOException, InterruptedException {
-        File dir = File.createTempFile("jenkins","git");
-        dir.delete();//result !needed
-        assertTrue(dir.mkdir());
+        File dir = Files.createTempDirectory("jenkins" + "git").toFile();//result !needed
+        assertTrue(true);
```

Comment:
```suggestion
        File dir = Files.createTempDirectory("jenkins" + "git").toFile();
```

remove the useless assertion and invalid comment